Tigran Kirakosov/flexbox_version/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>

```

Comment:
![grid_version](https://user-images.githubusercontent.com/15708576/61229058-2145c480-a730-11e9-9978-76fb4db011e4.png)
![flexbox_version](https://user-images.githubusercontent.com/15708576/61229060-2276f180-a730-11e9-95b9-7bb0206dbc56.png)
both versions didn't work in ie11

Reply:
and because you've already committed css files you need to delete it in new commit

Reply:
and about ie11 - firstly you can try to replace css variables to sass variables

Reply:
Ahh sure, I should've just deleted these files, right